mkdocs.yml

```diff
@@ -7,13 +7,29 @@ repo_url: https://github.com/aquasecurity/trivy
 edit_uri: ""
 
 nav:
-  - HOME: index.md
-  - Getting started:
-      - Overview: getting-started/overview.md
-      - Installation: getting-started/installation.md
-      - Quick Start: getting-started/quickstart.md
-      - Further Reading: getting-started/further.md
-  - Docs:
+  - Getting Started: 
+      - Overview: index.md
+      - Quick Start: home/quickstart.md
+      - Installation: home/installation.md
+  - Tutorials:
+      - Overview: tutorials/overview.md
+      - Trivy Kubernetes: 
+          - Trivy K8s: tutorials/kubernetes/trivy-kubernetes.md
```

Comment:
`Trivy K8s` is not clear. I think we should mention what users can do with this tutorial, e.g. "Cluster scanning" or something like that.

Reply:
maybe Kubernetes scanning

Reply:
ditto

Reply:
ditto